ui/src/components/config_section/ConfigSectionTitle.js

```diff
@@ -1,15 +1,16 @@
 import React from "react";
-import { EuiIcon, EuiTextColor, EuiTitle } from "@elastic/eui";
+import { EuiIcon, EuiTextColor, EuiTitle, EuiSpacer } from "@elastic/eui";
 
 export const ConfigSectionTitle = ({ title, iconType }) => (
   <EuiTitle size="s">
-    <EuiTextColor color="secondary">
+    <EuiTextColor color="#007871">
```

Comment:
This doesn't look right. Maybe you can take some inspiration from this: https://elastic.github.io/eui/#/theming/customizing-themes ?

Reply:
Based on the color code and the release notes, "secondary" is simply renamed as "success" where it was present: https://github.com/elastic/eui/releases/tag/v34.5.0

Reply:
Hmm something is overwriting the colour corresponding to `success` to black instead of `#00BFB3`. There're probably some styles in `mlp-ui` overwriting this.

Reply:
You're right - `EuiTitle` on its own is using a black-ish color which overrides what we set:

<img width="956" alt="Screenshot 2022-09-01 at 11 30 34 AM" src="https://user-images.githubusercontent.com/23465343/187825927-a41a8afa-82a3-49c2-bd47-e0da45bb55cf.png">

It works if we wrap the Text entity in `<span>`:
```html
 <EuiTitle size="s">
    <span>
      <EuiTextColor color="success">
```

In the previous Eui version, you can see the Euititle style had the black color which was correctly being overridden by the color we set:
<img width="975" alt="Screenshot 2022-09-01 at 11 37 38 AM" src="https://user-images.githubusercontent.com/23465343/187826687-d410a76f-3a72-47e0-908e-e8dd06303ba6.png">


But even with the `span` workaround, there is an unexpected outcome - the text is still using the old "secondary" value which isn't cheerful: https://elastic.github.io/eui/#/display/text#coloring-text

I believe that's still fine? (Wont hurt eyes as much lol.)
<img width="1072" alt="Screenshot 2022-09-01 at 11 34 25 AM" src="https://user-images.githubusercontent.com/23465343/187826312-64e1de0f-dea8-4a75-ab64-ab4e69e0cbaa.png">

Reply:
There is a similar discussion (although old) on the Eui repo, the suggestion seems to be to use a heading / span element inside the title: https://github.com/elastic/eui/issues/956

Reply:
Okay, so I guess this means we're going with the old `secondary` colour, and then letting `mlp-ui` do the upgrading of the styles to the new `success` colour :) I'll update all the instances where we're using `EuiTextColor` so that they'll use the old colour.